json-rpc/src/runtime.rs

```diff
@@ -150,7 +165,10 @@ pub fn bootstrap(
 
     let health_route = warp::path!("-" / "healthy")
         .and(warp::path::end())
-        .map(|| "diem-node:ok");
+        .and(warp::query().map(move |params: HealthCheckParams| params))
+        .and(warp::any().map(move || diem_db.clone()))
+        .and(warp::any().map(SystemTime::now))
```

Comment:
Oh, are we comparing to current time, not the latest block time? This isn't ideal in my opinion, since the chain itself losing consensus will show as lag on the fullnodes, when that's not the case. Is it not possible to know the latest block time?

Reply:
It also assumes that clocks are synced, which is not always the case.

Reply:
it checks: server latest ledger info timestamp >= server current time timestamp + duration_secs

Reply:
My understanding is that the ALB only supports a static URL to perform a query. This allows whomever is leveraging this service to have an appropriate tolerance that makes sense for their environments. You could theoretically set 60 seconds as your span and as long as the blockchain timestamp and your skew are within 60 seconds of each other, you'd be fine.

Once we get information about highest known LI in place, we can explore that methodology instead, but this is the best we currently have.